tests/test_core.py

```diff
@@ -7469,15 +7479,20 @@ def test_async_ccall_promise(self):
 ''')
     create_file('pre.js', r'''
 Module['onRuntimeInitialized'] = function() {
+  runtimeKeepalivePush();
```

Comment:
Why do we need to do this manually here?

Reply:
Because of this issue: https://github.com/emscripten-core/emscripten/pull/15085

I will last that first and revert this part.